crates/net/discv4/src/lib.rs

```diff
@@ -525,6 +525,8 @@ impl Discv4Service {
     pub fn spawn(mut self) -> JoinHandle<()> {
         tokio::task::spawn(async move {
             self.bootstrap();
+            self.add_all_nodes(self.config.trusted_nodes.clone());
+
```

Comment:
hmm, unsure about how this should be integrated with discovery. since these are different types of nodes, for example perhaps you don't want to announce your trusted nodes via discovery.

So imo should remove here and only use in peerset.